plugins/com.google.cloud.tools.eclipse.sdk.test/src/com/google/cloud/tools/eclipse/sdk/internal/CloudSdkProviderTest.java

```diff
@@ -48,4 +58,153 @@ public void testSetPreferenceInvalid() throws Exception {
       // ignore
     }
   }
+  
+  private static class MockPreferences implements IPreferenceStore{
```

Comment:
If Mockito works, it'd be much cleaner to use a `@Mock` instead of this lengthy default implementation.